bliss/predict.py

```diff
@@ -75,8 +76,14 @@ def predict(cfg: DictConfig):
     assert isinstance(bands, list) and len(bands) == 1, "Only 1 band supported"
 
     sdss_obj = sdss.SloanDigitalSkySurvey(**cfg.predict.sdss_kwargs)
-    sleep_net = SleepPhase.load_from_checkpoint(cfg.predict.sleep_checkpoint)
-    galaxy_encoder = GalaxyEncoder.load_from_checkpoint(cfg.predict.galaxy_checkpoint)
+    encoder = instantiate(cfg.model.encoder)
+    decoder = instantiate(cfg.model.decoder)
+    sleep_net = SleepPhase.load_from_checkpoint(
+        cfg.predict.sleep_checkpoint, encoder=encoder, decoder=decoder
+    )
+    galaxy_encoder = GalaxyEncoder.load_from_checkpoint(
+        cfg.predict.galaxy_checkpoint, decoder=decoder
+    )
```

Comment:
I see that now you need to instantiate the encoder/decoder separately when loading the checkpoint. The encoder that you pass in to `SleepPhase` is not trained, will the `sleep_net.image_encoder` be the trained version from the checkpoint?

Reply:
Could you also explain why you need to pass these two arguments at all? I would imagine that the checkpoint saved these models too, but maybe I'm missing something. Thanks!